src/playlist-controller.js

```diff
@@ -1059,15 +1059,10 @@ export class PlaylistController extends videojs.EventTarget {
 
   runFastQualitySwitch_() {
     this.waitingForFastQualityPlaylistReceived_ = false;
-    // Delete all buffered data to allow an immediate quality switch, then seek to give
-    // the browser a kick to remove any cached frames from the previous rendtion (.04 seconds
-    // ahead was roughly the minimum that will accomplish this across a variety of content
-    // in IE and Edge, but seeking in place is sufficient on all other browsers)
-    // Edge/IE bug: https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/14600375/
-    // Chrome bug: https://bugs.chromium.org/p/chromium/issues/detail?id=651904
+    // Delete all buffered data to allow an immediate quality switch.
     this.mainSegmentLoader_.pause();
     this.mainSegmentLoader_.resetEverything(() => {
-      this.tech_.setCurrentTime(this.tech_.currentTime());
+      this.mainSegmentLoader_.load();
```

Comment:
Do we want to load other loaders here?

Reply:
I don't think it's necessary, the only reason `load` was previously called on all the loaders was due to `setCurrentTime` calling `pause` then `resetEverything on _all_ the loaders, including the `mainSegmentLoader` a second time. https://github.com/videojs/http-streaming/blob/0b4da7ce70b49f0287600a0ef2f1468ec79e5e34/src/playlist-controller.js#L1535